string name
float64[] red_u
float64[] red_v
float64[] yellow_u
float64[] yellow_v
float64[] green_u
float64[] green_v
float64[] purple_u
float64[] purple_v
float64[] orange_u
float64[] orange_v
float64[] blue_u
float64[] blue_v